assets/karma.conf.js

```diff
@@ -85,7 +85,15 @@ module.exports = function (config) {
 
         // Concurrency level
         // how many browser should be started simultaneous
-        concurrency: Infinity
+        concurrency: Infinity,
+
+        // After upgrading to Jasmine 3.x, the tests fail when run in random
+        // order. Need to track down why this is the case.
+        client: {
+            jasmine: {
+                random: false
```

Comment:
Jasmine 3 will by default run tests in random order, which is surfacing a problem with our tests. I will create a ticket to track down why.